whisper/whisperv6/filter.go

```diff
@@ -85,6 +85,15 @@ func (fs *Filters) Install(watcher *Filter) (string, error) {
 	return id, err
 }
 
+// Each is a thread safe iterator over all existing filters.
+func (fs *Filters) Each(f func(filter *Filter)) {
```

Comment:
Either change all instances of `range fs.watchers` or do not use this function. I recommend not using it, because it implies creating an anonymous functions, and we both know what that entails in terms of "Javascript"-ability of the code. We strive to keep the Whisper code as dumb as possible, to ensure it is easy to audit. In any case, this should be a refactoring and not part of this PR.